logstash-core/lib/logstash/settings.rb

```diff
@@ -281,6 +281,10 @@ def to_hash
       }
     end
 
+    def inspect
```

Comment:
This makes failing expectations involving the `Setting` object a _lot_ easier to understand.

before:
~~~
expected `#<LogStash::Setting::PositiveInteger:0x1e48ac75 @name="pipeline.workers", @value=nil, @value_is_set=f...tor_proc=#<Proc:0x624fe3ee@/Users/yaauie/src/elastic/ls/logstash-core/lib/logstash/settings.rb:398>>.set?` to be truthy, got false
~~~

after:
~~~
expected `<LogStash::Setting::PositiveInteger(pipeline.workers): 16 (DEFAULT)>.set?` to be truthy, got false
~~~